src/QtLocationPlugin/QMLControl/QGCMapEngineManager.h

```diff
@@ -37,26 +37,27 @@ class QGCMapEngineManager : public QGCTool
     };
     Q_ENUM(ImportAction)
 
-    Q_PROPERTY(quint64              tileCount       READ    tileCount       NOTIFY tileCountChanged)
```

Comment:
please don't change indentation. Although it might not be one's preference (neither mine for example), let's not change it without a real reason. It's harder to read changes/merge the code.

Reply:
Sounds good to me, but just be aware that my change there will not align with the other Q_PROPERTYs.

If we want to keep everything aligned, I could throw my indentation changes in a separate commit.